packages/analytics-js-integrations/src/integrations/Braze/browser.js

```diff
@@ -74,18 +85,38 @@ class Braze {
     return window.brazeQueue === null;
   }
 
-  isReady() {
-    if (this.isLoaded()) {
-      try {
-        const anonymousId = this.analytics.getAnonymousId();
-        window.braze.getUser().addAlias(anonymousId, 'rudder_id');
-        return true;
-      } catch (error) {
-        logger.error(`Error in isReady - ${stringifyWithoutCircularV1(error, true)}`);
+  setUserAlias() {
+    try {
+      const anonymousId = this.analytics.getAnonymousId();
+      if (!anonymousId) {
```

Comment:
You can skip this case as anon ID will always be available.

Reply:
Keeping it for consistency :)